zqd/packet/packet.go

```diff
@@ -80,7 +83,10 @@ func (p *IngestProcess) run(ctx context.Context) error {
 	if err = p.space.SetPacketPath(p.pcapPath); err != nil {
 		goto abort
 	}
-	if err = p.writeData(ctx); err != nil {
+	if minTs, maxTs, err = p.writeData(ctx); err != nil {
+		goto abort
+	}
+	if err = p.space.SetTimes(minTs, maxTs); err != nil {
```

Comment:
Move this into writeData since the logic there will eventually do progressive updates to all.bnzg and we want a tight coupling between the zng file and the meta-data about the zng file.

Reply:
done